Resources/Prototypes/_CM14/Roles/Jobs/Marines/combat_tech.yml

```diff
@@ -30,6 +30,7 @@
   id: CMGearCombatTech
   equipment:
     jumpsuit: CMJumpsuitMarineEngineer # TODO CM14 all squad roles should spawn with no jumpsuit but still have clothes on
+    gloves: CMHandsInsulated
```

Comment:
This should be added to CMVendorBundleCombatTechnicianApparel, under ` bundle`, not given to them on spawn inside cryo. I was confused which one the issue meant, but it is the former.